### ISSUE TITLE
Feature Request / "Minor Code change Request" - Move SQLite database and config-file for persistent storage

### ISSUE THREAD

**Issue by dutch2005** [Ousret/hermes#50]
Request: Move SQLite database and config-file.

As can be found here: https://codefresh.io/docker-tutorial/build-docker-image-dockerfiles/

One can use "VOLUME" statements to make sure that the SQLite

Upon build one can define were the SQLite database is stored (so you can just pull a new docker and not lose your data)

I would like to suggest the SQLlite database gets moved 1 folder upwards "hermes.sqlite" currently resides in "/app/hermes_ui", moving it to the following folder "/app/hermes_ui/sqlite" would allow the statement in the dockerfile to keep the file "save".

If one does not wishes to use the "VOLUME" statement in the Dockerfile at the veryleasy upon startup you can make a mount between the used database (on the host) and the docker VM.

Something along the line of "docker run -v /app/hermes_ui/sqlite:/app/hermes_ui/sqlite" hermes

Once one makes a new SQLite it all gets stored / saved.

The same can be done with the config file (you dont want to keep on pulling a new docker and then to change the configuration.yml each time a new docker pull is made.

**Comment by Ousret** [comment]
Good call. We may want to create a distinct volume for DB, MariaDB, or SQLite. That's for sure.
We'll abandon the configuration file for ENV variables instead.

It's pretty easy to create volumes for DB, will do it asap.
